node/resolvers/search/index.ts

```diff
@@ -215,39 +247,67 @@ export const queries = {
       itemsReturned,
     }
   },
-
+  topSearches: () => {
+    return {
+      searches: [],
+    }
+  },
+  suggestionSearches: () => {
+    return {
+      searches: [],
+    }
+  },
+  suggestionProducts: () => {
+    return {
+      count: 0,
+      products: [],
+    }
+  },
```

Comment:
It would be nice to add comments here briefly explaining why these things return empty `searches` or `products`

Reply:
Nice suggestion!
I've added some comments explaining it.